packages/page-staking/src/Nomination/EraToTime.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useApi} from '@polkadot/react-hooks';
+import { useTranslation} from '@polkadot/app-accounts/translate';
+import { BareProps} from "@polkadot/react-api/types";
+import { BlockToTime} from "@polkadot/react-query";
+import { formatNumber } from '@polkadot/util';
+
+interface EraToTimeInterface extends BareProps{
```

Comment:
```suggestion
interface EraToTimeInterface extends BareProps {
```

Linting will pick this up.